### ISSUE TITLE
NOOB help. MMM-Pages appears to work but I wont rotate.... it does work exactly as programmed when clicking MMM-PageIndicator buttons with mouse.

### ISSUE THREAD

**Issue by Bernard-h** [edward-shen/MMM-pages#57]
Hi Guys thats for the support, and thanks for developing this modules Ed. I very much appreciate it. Can yourself (or any other more experienced user ) help me out with this simple problem I have.

When starting mirror npm start, All modules load properly. My homepage however is OVER-stacked with modules so I installed both modules (Pages and PageIndicator). I have added rotationtime but the pages wont change. However; when I mouse clock the pagebuttons on the page-indicator module, it works EXACTLY how I planned / designed the modules. 

- My MMM-Pages wont rotate pages. Its probably something simple. Herewith the most important parts or config.js:

{
		module: 'MMM-Todoist',
		position: 'bottom_right',	// This can be any of the regions. Best results in left or right regions.
		header: 'Shopping', // This is optional
		config: { // See 'Configuration options' for more information.
			hideWhenEmpty: true,
			accessToken: 'XXXXXXXXXXXX',
			maximumEntries: 10,
			showProject: false,
			updateInterval: 1*60*1000, // Update every 10 minutes
			fade: false,      
			// projects and/or labels is mandatory:
			projects: ['XXXXXXXX'], 
			labels: [ "MagicMirror", "Important" ] // Tasks for any projects with these labels will be shown.
      }
	},

	

{	
			module: 'nstreinen',
    			position: 'top_left',
    			header: 'Rotterdam CS naar Laan vNOI',
    			config: {
      			apiKey:'XXXXXXXXXXX',
      			fromStation: 'RTD',
			toStation: 'LAA',
			maxEntries: 2
    			}
		},

// letop jaarlijks de config file handmatig updaten met behulp van de afvalkalander van de gemeente.

{
  module: 'MMM-MyGarbage',
  position: 'top_right',
  header: 'Container',
  config: {
          alert: 4,
          weeksToDisplay: 1,
          limitTo: 2,
          fade: false,
          dateFormat: "dddd D MMMM",
          fadePoint: 0.25
  }
},

 

{
		module: "MMM-MotionDetector",
//		position: "top_left",	// Optional. This can be any of the regions. Displays debug informations.
		config: {
				captureIntervalTime: 1000,
				scoreThreshold: 150,
				timeout: 120000
			// The config property is optional.
			// See 'Configuration options' for more information.
		}
	},
	
//		{
//			module: "compliments",
//			position: "lower_third"
//		},

// niet zo boeiend dit is de standaard weather display app. per nu uitgeschakeld en geupgrade 
//
//		{
//			module: "currentweather",
//			position: "top_left",
//			config: {
//				location: "XXXXXXX",
//				locationID: "XXXXXX", //ID from http://bulk.openweathermap.org/sample/city.list.json.gz; unzip the gz file and find your city
//				appid: "XXXXXXXXXXXX"
//				onlyTemp: "true",
//			}
//		},

//		{
//			module: "weatherforecast",
//			position: "top_right",
//			header: "Weather Forecast",
//			config: {
//				location: "New York",
//				locationID: "5128581", //ID from http://bulk.openweathermap.org/sample/city.list.json.gz; unzip the gz file and find your city
//				appid: "YOUR_OPENWEATHER_API_KEY"
//			}
//		},
		{
		module: "newsfeed",
		position: "top_bar",	 
		config: {
		hideLoading: true, 
		ignoreOldItems: true,
		ignoreOlderThan: 6000000,
//			// The config property is optional.
//			// If no config is set, an example calendar is shown.
//			// See 'Configuration options' for more information.

			feeds: [
				{
					title: "NL Alert / Rijnmond veiligheidsregio",
					url: "https://rss.rijnmondveilig.nl/rotterdamrijnmond/index.rss",
				},
				{
					title: "NL Alert / Rijnmond veiligheidsregio",
					url: "https://crisis.nl/algemene-informatie-van-crisisnl/",
				},

			]
		},
	},

{
    module: "MMM-NLDepartureTimes",
    position: "top_left",
    header: "Randstadrail",
    config: {
      maxVehics: 4,
      updateSpeed: 8,
      locale: 'nl-NL',
      tpc: {
        '':{
          '': [31008713, 31008712],
        },
      }
    }	
},

		{
		module: "newsfeed",
		position: "top_bar",	 
		config: {
		hideLoading: true, 
		updateInterval: 5000,
			// The config property is optional.
			// If no config is set, an example calendar is shown.
			// See 'Configuration options' for more information.

			feeds: [
				{
					title: "NU",
					url: "http://www.nu.nl/rss/Algemeen",
				},
				{
					title: "RTE",
					url: "https://www.rte.ie/feeds/rss/?index=/news/&limit=20",
				},
//				{
//					title: "Leonidas",
//					url: "https://twitrss.me/twitter_user_to_rss/?user=@RKSV_Leonidas",
//				},
				{
					title: "Sky News",
					url: "http://feeds.skynews.com/feeds/rss/world.xml",
				},
//				{
//					title: "Rijnmond",
//					url: "https://wwww.rijnmond.nl/rss",
//				},
			]
		},
	},
	
		{
        module: 'MMM-pages',
        config: {
                modules:
                    [[ "MMM-NLDepartureTimes", "MMM-Todoist" ],
                     [ "nstreinen", "MMM-MyScoreboard" ]],
                fixed: ["updatenotification", "alert", "calendar", "weatherforecast", "MMM-ImagesPhotos", "MMM-Pages", "MMM-NowplayingOnSpotify", "MMM-rainfc", "MMM-Screencast", " MMM-MotionDetector", "MMM-MyGarbage", "clock", "currentweather", "MMM-page-indicator", "newsfeed", "MMM-OpenmapWeather",],
		rotationTime: 10000,
		homePage: 1,
		
        }
    },

	]

};
		

/*************** DO NOT EDIT THE LINE BELOW ***************/
if (typeof module !== "undefined") {module.exports = config;}

**Comment by edward-shen** [comment]
Please attach your full config--the one you've provided appears to be invalid.

**Comment by Bernard-h** [comment]
My apologies, herewith the full config.js from start to finish:

/* Magic Mirror Config Sample
 *
 * By Michael Teeuw http://michaelteeuw.nl
 * MIT Licensed.
 *
 * For more information on how you can configure this file
 * See https://github.com/MichMich/MagicMirror#configuration
 *
 */

var config = {
	address: "localhost", // Address to listen on, can be:
	                      // - "localhost", "127.0.0.1", "::1" to listen on loopback interface
	                      // - another specific IPv4/6 to listen on a specific interface
	                      // - "0.0.0.0", "::" to listen on any interface
	                      // Default, when address config is left out or empty, is "localhost"
	port: 8080,
	ipWhitelist: ["127.0.0.1", "::ffff:127.0.0.1", "::1"], // Set [] to allow all IP addresses
	                                                       // or add a specific IPv4 of 192.168.1.5 :
	                                                       // ["127.0.0.1", "::ffff:127.0.0.1", "::1", "::ffff:192.168.1.5"],
	                                                       // or IPv4 range of 192.168.3.0 --> 192.168.3.15 use CIDR format :
	                                                       // ["127.0.0.1", "::ffff:127.0.0.1", "::1", "::ffff:192.168.3.0/28"],

	useHttps: false, 		// Support HTTPS or not, default "false" will use HTTP
	httpsPrivateKey: "", 	// HTTPS private key path, only require when useHttps is true
	httpsCertificate: "", 	// HTTPS Certificate path, only require when useHttps is true

	language: "en",
	timeFormat: 24,
	units: "metric",
	// serverOnly:  true/false/"local" ,
			     // local for armv6l processors, default
			     //   starts serveronly and then starts chrome browser
			     // false, default for all  NON-armv6l devices
			     // true, force serveronly mode, because you want to.. no UI on this device



	modules: [
		
		

		
		
		{
			module: "alert",
		},
		{
			module: "updatenotification",
			position: "top_bar"
		},
		{
			module: "clock",
			position: "top_left"
		},
		{
			module: "calendar",
			header: "Kalender",
			position: "top_left",
			config:{
			maximumEntries: 5,
			maxTitleLines: 20,
			calendars: [
			{
				symbol: "My Calendar",
				url: "XXXXX"
			}
			]
		}
	},
	
	

// deze module trekt nu gifjes uit een folder van de module! updaten handmatig (eventueel in de toekomst een netwerkadres invoeren voor familie/vakantie/personal shit	

{
	module: "MMM-ImagesPhotos",
	position: "bottom_left",
	config: {
		opacity: 0.9,
		animationSpeed: 1000,
		updateInterval: 12000,
		maxWidth: "350px",
		maxHeight: "350px",
	}
},

//weinig ruimte op de mirror maar je kan de album art aanzetten

{
  module: "MMM-NowPlayingOnSpotify",
  position: "bottom_left",

  config: {
    showCoverArt: false,  
    clientID: "XXXXX",
    clientSecret: "XXXXXX",
    accessToken: "XXXX",
    refreshToken: "XXXX"
  }
},

{
        module: 'MMM-pages',
        config: {
                modules:
                    [[ "MMM-NLDepartureTimes", "MMM-Todoist" ],
                     [ "nstreinen", "MMM-MyScoreboard" ]],
                fixed: [ "alert", "calendar", "weatherforecast", "MMM-ImagesPhotos", "MMM-Pages", "MMM-NowplayingOnSpotify", "MMM-rainfc", "MMM-Screencast", "MMM-MotionDetector", "MMM-MyGarbage", "clock", "currentweather", "MMM-page-indicator", "newsfeed", "MMM-OpenmapWeather"],
		rotationTime: 10000,
        }
    },

		

// voor specifieker filters, even goed kijken naar de filters en de faq op github. Er zjin per jaar enkele dagen waarin te veel wedstrijden zijn (denk aan 1/2/3e ronde beker spanje en nederland. In dat geval wordt de display te groot en gaat hij andere modules overlappen. Het gaat om 6-7 x per jaar, dus accepteren kan ook.

{
  module: "MMM-MyScoreboard",
  position: "bottom_right",
  classes: "default everyone",
  header: "Uitslagen",
  config: {
    showLeagueSeparators: true,
    colored: true,
    rolloverHours: 3,
    viewStyle: "smallLogos",
    sports: [
	{
        league: "ESP_COPA_DEL_REY",
	label: "Copa Del Rey"        
      },      
	{
        league: "NED_EREDIVISIE",
	label: "Eredivisie"        
      },
      {
        league: "UEFA_EUROPE_QUAL",
	label: "EL kwalificatie"        
      },
      {
        league: "UEFA_EUROPA",
	label: "EL"        
      },
      {
        league: "NED_KNVB_BEKER",
	label: "Beker"        
      },
      {
        league: "UEFA_EUROPA",
	teams: ["AJAX", "PSV", "FEYE", "VIT", "UTR", "ALK"],
	label: "EL"        
      },
	{
        league: "UEFA_CHAMPIONS",
//	teams: ["AJAX", "PSV", "FEYE", "VIT", "UTR", "ALK"],
	label: "CL"        
      },
      {
        league: "UEFA_EUROPEAN_CHAMPIONSHIP",
	label: "EK"        
      },
      {
        league: "FIFA_WORLD_CUP",
	label: "WK"        
},
      
      
    ]

  }
 },









{
		module: "MMM-OpenmapWeather",
		position: "top_right",	// This can be any of the regions.
									// Best results in left or right regions.
		config: {
			// See 'Configuration options' for more information.
			location: "Berkel en Rodenrijs",
			locationID: "XXXXXX", //Location ID from http://openweathermap.org/help/city_list.txt
			appid: "XXXXX",  //openweathermap.org API key
      			colorIcon: true ,
			showWindDirectionAsArrow: true ,
			showFeelsLike: false ,
			roundTemp: true ,
			showSunset: false 
			
		}
	},



 
 
 {
			module: "weatherforecast",
			position: "top_right",
			header: "Weather Forecast",
			config: {
				location: "XXXX",
				locationID: "XXXX",  //ID from http://bulk.openweathermap.org/sample/city.list.json.gz; unzip the gz file and find your city
				appid: "XXXXXX,
				maxNumberOfDays: 5,
				roundTemp: true
				
			}
		},

{
		module: "MMM-rainfc",
		position: "top_right",
            	header: "Buienradar",
		config: {
			lat: "51.99",
			lon: "4.46", 
			width: 150,
			height: 45,
			lineWidth: 2,
			lineColor: "#e0ffe0",
			fillColor: "#e0ffe0",
			maxPower: 5,
			rainText: "Tot: ",
			noRainText: "Geen regen tot: ",
			nrOfTimeLabels: 5 // advised values: 2-5
		}
	},

//     handig maar lelijk, heb nu een kleine zakelijke neerslag module rechts. Dat is beter. Deze module in de backup houden, misschien leuk als er meerdere paginas komen
//
//		{
//            module: 'MMM-Buienradar',
//            position: 'bottom_left',
//            config: {
//              lat: 51.953326,
//              lon: 4.465167,
//             forecast: true,
//              zoom: 4,
//              interval: 10
//           }
//        },



// module werkt maar sinds de herinstallatie 20-6-20 met diverse geluidsissues (raspotify / alexa /screencast /cq reden voor herinstallatie) krijg ik deze module niet met audio, video werkt. Hier moet je je verdiepen in github resolves. Ik kan nu de juiste soundconfigs niet vinden en op fora blijkt dat na de raspbean update in maart 2020 er diverse soundissues zijn (zie ook folder home/pi/onderhoud voor textbeschrijvingen). Wellicht in de toekomst geluidsissue op te lossen. voor nu video only


{
		module: 'MMM-Screencast',
		position: 'center', // This position is for a hidden <div /> and not the screencast window
		config: {
			position: 'center',
			height: 600,
			width: 1000,
			castName: 'Mirror', 
		}
        },

{
        module: 'MMM-page-indicator',
        position: 'bottom_bar',
        config: {
            pages: 2,
        }
    },



{
		module: 'MMM-Todoist',
		position: 'bottom_right',	// This can be any of the regions. Best results in left or right regions.
		header: 'Shopping', // This is optional
		config: { // See 'Configuration options' for more information.
			hideWhenEmpty: true,
			accessToken: 'XXXXX',
			maximumEntries: 10,
			showProject: false,
			updateInterval: 1*60*1000, // Update every 10 minutes
			fade: false,      
			// projects and/or labels is mandatory:
			projects: ['XXXXX], 
			labels: [ "MagicMirror", "Important" ] // Tasks for any projects with these labels will be shown.
      }
	},

	

{	
			module: 'nstreinen',
    			position: 'top_left',
    			header: 'Rotterdam CS naar Laan vNOI',
    			config: {
      			apiKey:'XXXXX',
      			fromStation: 'RTD',
			toStation: 'LAA',
			maxEntries: 2
    			}
		},

// letop jaarlijks de config file handmatig updaten met behulp van de afvalkalander van de gemeente.

{
  module: 'MMM-MyGarbage',
  position: 'top_right',
  header: 'Container',
  config: {
          alert: 4,
          weeksToDisplay: 1,
          limitTo: 2,
          fade: false,
          dateFormat: "dddd D MMMM",
          fadePoint: 0.25
  }
},


 


// de position regel is uitgeschakeld, als je de motion wil meten en aan het troubleshooten bent met de motion dectotion moet je deze regel activeren. Dan zie je de down-time up-time en gemeten pixeldifference.


{
		module: "MMM-MotionDetector",
//		position: "top_left",	// Optional. This can be any of the regions. Displays debug informations.
		config: {
				captureIntervalTime: 1000,
				scoreThreshold: 150,
				timeout: 120000
			// The config property is optional.
			// See 'Configuration options' for more information.
		}
	},

	
//		{
//			module: "compliments",
//			position: "lower_third"
//		},


// niet zo boeiend dit is de standaard weather display app. per nu uitgeschakeld en geupgrade 
//
//		{
//			module: "currentweather",
//			position: "top_left",
//			config: {
//				location: "XX",
//				locationID: "XX", //ID from http://bulk.openweathermap.org/sample/city.list.json.gz; unzip the gz file and find your city
//				appid: "XX"
//				onlyTemp: "true",
//			}
//		},


//		{
//			module: "weatherforecast",
//			position: "top_right",
//			header: "Weather Forecast",
//			config: {
//				location: "New York",
//				locationID: "5128581", //ID from http://bulk.openweathermap.org/sample/city.list.json.gz; unzip the gz file and find your city
//				appid: "YOUR_OPENWEATHER_API_KEY"
//			}
//		},
		
		
		
{
		module: "newsfeed",
		position: "bottom_bar",	// This can be any of the regions. Best results in center regions.
		config: {
		showSourceTitle: false,
		updateInterval: 10000,
		animationSpeed: 1000,
		showDescription: false,
			// The config property is optional.
			// If no config is set, an example calendar is shown.
			// See 'Configuration options' for more information.

			feeds: [
				{
					title: "New York Times",
					url: "http://www.nytimes.com/services/xml/rss/nyt/HomePage.xml",
				},
				{
					title: "BBC",
					url: "http://feeds.bbci.co.uk/news/video_and_audio/news_front_page/rss.xml?edition=uk",
				},
			]
		}
	},

		

{
    module: "MMM-NLDepartureTimes",
    position: "top_left",
    header: "Randstadrail",
    config: {
      maxVehics: 4,
      updateSpeed: 8,
      locale: 'nl-NL',
      tpc: {
        '':{
          '': [31008713, 31008712],
        },
      }
    }	
},





		
		

	]

};
		


/*************** DO NOT EDIT THE LINE BELOW ***************/
if (typeof module !== "undefined") {module.exports = config;}

**Comment by edward-shen** [comment]
Your config is still invalid, which could explain the wierdness. For example, your weather forecast config is missing a " on the `appid` line, and the MMM-Todoist config is missing a ' on the `projects` line.

**Comment by Bernard-h** [comment]
Hi Edward , thanks for coming back to me. I just pasted these lines wrong. I reverted back to an old backup, installed the mod and it works fine! However the problem is not MMM-Pages, nor the module MMM-Todoist. Both modules work, but these modules f- each other up. I think its some error MMM-Todoist is causing. I had MMM-Pages working until 5 minutes ago when MMM-Todoist f-ed it all up. I consider this issue closed since the mod works. Below the config output when installing Todoist for reference.


![Schermafbeelding 2021-03-17 om 21 20 55](https://user-images.githubusercontent.com/60395872/111533458-1514d480-8767-11eb-8966-a46a6744a8e7.png)
